doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -650,6 +652,15 @@ Matter samples
   * Removed support for nRF54H20 devices.
   * Updated the API of ``AppTask``, ``BoltLockManager``, and ``AccessManager`` to provide additional information for the ``LockOperation`` event.
 
+* Updated all Matter samples that runs Wi-Fi stack to store the Wi-Fi firmware patch in external flash.
```

Comment:
Suggesting that you place this entry under the first Added bullet.

Reply:
ok, moved to the `Updated` bullet.